Custom-Audio-Device/README.md

```diff
@@ -195,20 +190,53 @@ static otk_thread_func_return_type capturer_thread_start_function(void *arg) {
 }
 ```
 
-#### How to set it
+The `otc_audio_device_write_capture_data()` function adds the audio byte
+array to the audio buffer that will be used for audio in the published stream.
+
+### Setting the audio device to be used by the application
 
 Once the application initializes the `struct audio_device` struct and sets the
-pointers to the callback functions the custom audio device can be set by
+pointers to the callback functions, the custom audio device can be set by
 using the `otc_set_audio_device()` function defined in the OpenTok Linux SDK.
 
 ```c
   otc_set_audio_device(&(device->audio_device_callbacks));
 ```
 
-As you see the function take a pointer to the `otc_audio_device_callbacks audio_device_callbacks`
-member in the `struct audio_device` struct.
+The function takes a pointer to the `audio_device_callbacks` struct
+(of type `otc_audio_device_callbacks`) defined in the previous section.
+
+### Configuring the audio device
+
+After setting the audio device, the app calls the `audio_device_get_capture_settings()`
+function, defined by the OpenTok Linux SDK, to adjust configuration settings for the
+audio capturer:
+ 
```

Comment:
This does not work this way actually. At some point the SDK invokes the `get_capture_settings` function callback in the `otc_audio_device_callbacks` struct. The function bellow is the one we set for the `get_capture_settings` function pointer. When it gets invoked by the SDK the SDK is provided with the settings for the custom audio device being implemented in this sample.